.github/workflows/devDeployment.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  build:
+    name: Run ESLint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
```

Comment:
Qu'est-ce qui t'a fait choisir de passer par cette github action plutot que par les commandes direct d'heroku ?

Reply:
j'ai suivi cette doc : https://github.com/marketplace/actions/deploy-to-heroku

Reply:
Les github actions que l'on trouve sur le marketplace sont un peu comme les paquets que l'on installe sur npm. 
Du coup je me dis qu'on peut se poser les memes questions quand on en utilise une, qu'en penses-tu ? Notamment dans ce paquet, la derniere mise à jour date de 2021 et il y a pas mal d'issues ouverte. 

Pour info, il me semble que pour l'instant dans les paquets qu'on utilise dans les autres actions, il n'y a que des actions soit faites directement par github soit par un auteur certifié (cypress).